### PR TITLE
Add liquid box creation and simulation files

### PR DESCRIPTION
**Pull request by leeping**
This PR adds files and scripts for creating a liquid box and running a SMIRNOFF simulation on it, starting from just a SMILES string for a single molecule of the liquid.  Example included for ethanol.

### CODE REVIEW

**Comment by hjuinj** [comment]
I also wrote somethings similar using the python packmol handles John developed in `openmoltools` [here](https://github.com/hjuinj/mdfptools/blob/backup/mdfptools/Parameteriser.py#L243-#L268)

**Comment by davidlmobley** [comment]
@hjuinj - yeah, I added those (or at least some of them) for use in solvationtoolkit (github.com/mobleylab/solvationtoolkit). I haven't extended that for  SMIRNOFF yet but I should.

File: examples/liquid_box/create_mol2_pdb/README.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+Provide a text file containing a single SMILES string and three-letter residue name, as:
+`python create_mol2_pdb.py input.smi ETH`
+Receive `ETH.pdb` and `ETH.mol2` files containing a single molecule with 1 conformation.
+Receive `ETH-box.pdb` containing a box with specified number of molecules.
+
+## Dependencies:
+- OpenEye tools (for creating molecule from SMILES and conformer generation)
+- openmoltools (OpenEye conformer generation wrapper)
+- Gromacs 4.6.7 (for calling genbox to create solvent box)
+- ForceBalance 1.5.x (for putting information back that was thrown away by genbox)
```
Comment (davidlmobley) [inline examples/liquid_box/create_mol2_pdb/README.md]: Not ideal to have ForceBalance and GROMACS dependencies, but probably OK for the time being.

**Comment by davidlmobley** [comment]
Will merge once @leeping acknowledges.

**Comment by leeping** [comment]
I agree with all of your comments and we can proceed with the merge.  There are certainly ways to accomplish the same thing with fewer dependencies that we should add in the future.

**Pull request merged**